(1) feat: Add Feedback Form Component (#4328)
getsentry/sentry-react-native: CHANGELOG.md

  });
  ```

- Adds feedback form ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))

  You can add the form component in your UI and customise it like:
  ```jsx
  import { FeedbackFormScreen } from "@sentry/react-native";
  ...
  <FeedbackFormScreen
    {...props}
    closeScreen={props.navigation.goBack}
    styles={{
      submitButton: {
        backgroundColor: '#6a1b9a',
        paddingVertical: 15,
        borderRadius: 5,
        alignItems: 'center',
        marginBottom: 10,
      },
    }}
    text={{namePlaceholder: 'Fullname'}}
  />
  ```

>   Check [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/) for more configuration options.

Author: The documentation will be updated with https://github.com/getsentry/sentry-docs/pull/11980 (WIP)

  });
  ```

- Adds feedback form ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))

  You can add the form component in your UI and customise it like:
  ```jsx
  import { FeedbackFormScreen } from "@sentry/react-native";
  ...

>   <FeedbackFormScreen

Reviewer: Let's just how the min config in the change log, so ideally with no props. (If we can do that, see my other comments.)

Author: Updated the changelog with 4b290a219787b50e5cc55c220b4fc25a98ee1dd5

  });
  ```

> - Adds feedback form (beta) ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))

Reviewer: ````suggestion
- User Feedback From Component Beta ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))
````

>   You can add the form component in your UI like:

Reviewer: ````suggestion
   To collect user feedback from inside your application add the `FeedbackFrom` component.

````

Author: Thank you for the better wording 🙇

  ```jsx
  import { FeedbackForm } from "@sentry/react-native";
  ...
  <FeedbackForm/>
  ```

- Export `Span` type from `@sentry/types` ([#4345](https://github.com/getsentry/sentry-react-native/pull/4345))

> 

Reviewer: ````suggestion
````

  });
  ```

- Adds feedback form (beta) ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))

  You can add the form component in your UI like:
  ```jsx
  import { FeedbackForm } from "@sentry/react-native";
  ...
  <FeedbackForm/>
  ```

>   Check [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/) for more configuration options.

Reviewer: Maybe we should remove the link to the docs until we updated them and move this out of beta

Author: Sounds good. Removed with 4097347e2b44db52c007a693aa9ee0efd004ca40

I'll update the [documentation PR](https://github.com/getsentry/sentry-docs/pull/11980/) as along with finalising the changes on the feedback form PRs.

  To learn how to attach context data to the feedback visit [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/).

> - User Feedback Form Component Beta ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))

Author: Fixed with 9853630c6efd07354cf608d1a19774a1d7b0cd9b
